### PR TITLE
Word places separated by spaces in final display. Tests corrected to …

### LINKED ISSUE #2: Improve the UI so that the placeholders indicate how many letters the word is made of.
Currently it displays like so for a 4 letter word:
```ruby
____
```
Should be more like:
```ruby
_ _ _ _
```

### PR DESCRIPTION
**Pull request by Katerina-codes**
…reflect this.

Fixes #2 
@Maikon

### CODE REVIEW

File: spec/game_flow_spec.rb [hunk 1]
```diff
@@ -17,6 +17,6 @@
     display = Display.new(output, input)
     game_flow = GameFlow.new(display)
     game_flow.game_flow("tan")
-    expect(output.string).to include("tan")
+    expect(output.string).to include("t a n")
```
Comment (Maikon) [inline spec/game_flow_spec.rb]: So will the end result show like this:
```
t a n
```
?

Comment (Katerina-codes) [reply spec/game_flow_spec.rb]: Yes I did intend that. I hadn't thought that maybe that could be unclear but I suppose it is. I will change it.

Comment (Maikon) [reply spec/game_flow_spec.rb]: Ok. Will it be part of this PR?

File: lib/display.rb [hunk 2]
```diff
@@ -51,4 +51,10 @@ def display_you_win
     @output.puts "You win! Woohoo!"
   end
 
+  def display_original_word(word)
+    word = word.delete(" ")
+    @output.puts word
+    word
```
Comment (Maikon) [inline lib/display.rb]: Is this line needed here?

Comment (Katerina-codes) [reply lib/display.rb]: Fixed.

**Review by Maikon — APPROVED** [review]

**Pull request merged**